Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityInputCapabilities.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Platform input capabilities.
+    /// </summary>
+    public enum MixedRealityInputCapabilities
```

Comment:
Also, consider naming it not input specific, based on @provencher's request below for having spatial awareness (i.e. they are device/system capabilities, not just input potentially)